float32 x
float32 y
uint16 easyProb # These need to be deleted ***
uint16 medProb # ***
uint16 hardProb # ***
uint16 terrainHazard
float32 cachedProb
float32 purpleProb
float32 redProb
float32 blueProb
float32 silverProb
float32 brassProb
bool searchable
